package/upgrade/migrations/upgrade_manifests/v1.4.0/pre-hook.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -ex
```

Comment:
I suggest not bailing out if these two patches are errors. The upgrade should continue, and it's OK to apply the patch manually

Reply:
Thanks, updated.